pkg/scheduler/schedule_one.go

```diff
@@ -152,15 +154,19 @@ func (sched *Scheduler) schedulingCycle(
 					nominatingInfo = result.NominatingInfo
 				}
 			}
+			status = framework.NewStatus(framework.Unschedulable, err.Error()).WithError(err)
```

Comment:
This just adds unnecessary conversions to string on production code.

```suggestion
			status = framework.NewStatus(framework.Unschedulable).WithError(err)
```